torch/ao/quantization/fx/backend_config/native.py

```diff
@@ -92,16 +96,205 @@ def _get_default_op_backend_config(op, dtype_configs):
         torch.nn.functional.layer_norm,
     ]]
 
-def get_native_backend_config_dict():
-    """ Get backend for PyTorch Native backend_config_dict (fbgemm/qnnpack)
+def _get_linear_configs():
     """
+    Return all configs related to linear modules and ops.
+    """
+    observation_type = ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT
+    dtype_configs = [weighted_op_int8_dtype_config]
+    linear_configs = []
+
+    # linear module
```

Comment:
can we classify (by adding comments) these configs a bit to make sure we have more structured support,
e.g.
1. single linear
2. linear + relu
3. linear + bn